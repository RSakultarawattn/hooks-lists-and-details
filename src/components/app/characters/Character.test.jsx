import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Character from './Character';

describe('displays a character', () => {
  afterEach(() => cleanup());
  it('renders one single character', () => {
    const { asFragment } = render(<Character image="" name="" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
