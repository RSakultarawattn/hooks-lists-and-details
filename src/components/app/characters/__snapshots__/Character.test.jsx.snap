// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays a character renders one single character 1`] = `
<DocumentFragment>
  <figure>
    <img
      src=""
    />
    <figcaption />
  </figure>
</DocumentFragment>
`;
